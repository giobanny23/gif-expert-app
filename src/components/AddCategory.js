import React, { useState } from 'react'
export const AddCategory = () => {


    const [inputValue, setInputValue] = useState('Hola Mundo')

    const handleInputChange = (e) => {
        setInputValue( e.target.value );
    }
    return (
        <>
            <h1>{ inputValue }</h1>
            <input
                type ="text"
                value = {inputValue}
                onChange ={ handleInputChange }
            
            />
        </>
    )
}
