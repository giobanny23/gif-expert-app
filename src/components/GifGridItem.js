import React from 'react'

export default function GifGridItem( img) {
    console.log(img)

    return (
        <div>

        </div>
    )
}
