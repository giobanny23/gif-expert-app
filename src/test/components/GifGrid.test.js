import React from 'react'
import { shallow} from 'enzyme';
import { GifGrid } from '../../components/GifGrid';

describe('Pruebas en el <GifGrid />', () => {

    const category = 'One Punch';
    
    test('debe mostrar correctamente', () => {

        const wrapper = shallow( <GifGrid category = {category}/>);
        console.log(GifGrid);
    })
    
})
