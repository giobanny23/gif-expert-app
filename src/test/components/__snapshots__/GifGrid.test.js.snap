// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el <GifGrid /> debe mostrar correctamente 1`] = `
<Fragment>
  <h3>
     
    One Punch
     
  </h3>
  <p>
     Loading
  </p>
  <div
    className="card-grid"
  />
</Fragment>
`;
